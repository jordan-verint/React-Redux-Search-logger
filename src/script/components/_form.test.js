import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import FormSearch from './_form';

describe("<FormSearch />", () => {
    it("should render correctly", () => {
        const output = shallow(<FormSearch title="mockTitle" url="mockUrl" />);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});