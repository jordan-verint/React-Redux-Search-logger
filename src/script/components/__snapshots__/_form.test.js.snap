// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormSearch /> should render correctly 1`] = `
<form
  action="?"
  onSubmit={[Function]}
>
  <div
    className="field search"
  >
    <input
      onChange={[Function]}
      type="text"
      value=""
    />
  </div>
  <div
    className="field"
  >
    <button>
      Submit
    </button>
  </div>
</form>
`;
